crates/next-core/js/src/read-config/next.ts

```diff
@@ -0,0 +1,45 @@
+import IPC, { Ipc } from "@vercel/turbopack-next/internal/ipc";
+import loadConfig from "next/dist/server/config";
+import { PHASE_DEVELOPMENT_SERVER } from "next/dist/shared/lib/constants";
+
+type IpcIncomingMessage = {
+  type: "loadNextConfig" | "loadPostcssConfig";
+  path: string;
+};
+
+type IpcOutgoingMessage = {
+  type: "javaScriptValue";
+  data: number[];
+};
+
+const ipc = IPC as Ipc<IpcIncomingMessage, IpcOutgoingMessage>;
+
+const CACHED_CONFIGS = new Map<string, any>();
```

Comment:
I think we're better off caching this on the rust side with turbo tasks?

Also this cache is never used